### DISCUSSION TITLE
Как будет реализован приват?

### DISCUSSION THREAD

**Discussion by So0vey** [TerraFirmaGreg-Team/TFG-Modpack-1.20.x#173] [Ideas]
### Версия используемого модпака

Последняя

### Связанная проблема

На данный момент на сервере отсутствует лимит чанков на приват. (Ну тоесть их 500, что очень много)
Неужели как на 1.12 будет крошечный лимит, а когда админа попросят, он лимит поднимет?

### Ваше решение

Я предлагаю несколько вариантов.
1. Как в тауни. Сначала чанков мало, но чем больше людей в пачке, тем больше чанков доступно. (Проблема: бесконечные твинки в пачке. - Решение: жестко ограничить количество твинков. Вплоть до бана)
2.  За выполнение квестов и переход на новую эпоху развития, будет добавляться новые доступные чанки. Избавит от проблемы с тем, что человек со старта сможет заприватить все и сразу. А так хоть пару часов поиграть придеться.
3. Ничего не менять, и оставить так, как это было на 1.12.
4. Сделать количество чанков компромиссным. Новичку будет их много, но пол карты заприватить не сможет. Старичку будет не хватать чанков.
5. Объединить пункты 1 и 2 вместе.

З.ы. И обязательно нужно сделать так, чтобы приват был только неразрывным. В пределах одной планеты естественно.

**Comment by Latom-alex** [comment]
Я склоняюсь ко 2

**Comment by Exzept1on** [comment]
Звучит хорошо, будем делать в будущем, пункт 1 + 2.
Сейчас (в 0.0.6) уже работает 1.
